projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -54,11 +54,22 @@
       display: flex;
       width: 100%;
       align-items: center;
-      padding-left: 8px;
 
       .trigger-label {
         @include body-2-medium();
         padding-left: 8px;
+        width: calc(100% - 32px - 38px);
+        flex-grow: 1;
+      }
+
+      .trigger-more-items {
+        padding: 0 4px;
+        background-color: $gray-2;
+        border-radius: 4px;
+        margin-left: 8px;
+        max-width: 38px;
+        font-size: 12px;
+        font-weight: 600;
```

Comment:
can we pull this font from our shared font styles? We generally want all fonts to be defined in font.scss so we're not defining them adhoc in every style sheet.

Reply:
updated